public/style.scss

```diff
@@ -0,0 +1,86 @@
+/************************* Variables *************************/
+// all the global variables are defined here
+$correct: green;
+$green-shadow: rgb(156, 245, 156);
+$wrong: red;
+$red-shadow: rgb(253, 154, 154);
+/************************* Variables *************************/
+
+/************************* Global *************************/
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.centered {
+  position: fixed;
+  width: 80%;
+  max-width: 900px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  text-align: center;
+}
+
+.scrollable {
+  height: auto;
+  max-height: 80vh;
+  overflow: auto;
+}
+h1 {
+  font-size: 2rem;
+  margin: 1rem;
+}
+
+.correct {
+  border: 2px solid $correct;
+  box-shadow: 0 0 2px 1px $green-shadow;
+}
+
+.wrong {
+  border: 2px solid $wrong;
+  box-shadow: 0 0 2px 1px $red-shadow;
+}
+
+/************************* Desktop View *************************/
+ul {
```

Comment:
I wouldn't prefer that you use direct element names `ul, li, button` unless you are creating these styles for micro/basic component!.